src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1250,10 +1258,17 @@ private function discoverNamespace(\DOMXPath $domxpath, string $prefix): ?string
             return $this->namespaces[$prefix];
         }
 
+        if (array_key_exists($prefix, $this->cachedNamespaces)) {
```

Comment:
```suggestion
        if (\array_key_exists($prefix, $this->cachedNamespaces)) {
```

To make fabbot happy. 😃

Reply:
Just noticed, thanks.